src/services/near-nep21-util.js

```diff
@@ -174,9 +197,9 @@ export function trimZeros(str) {
  * normalizeAmount 135  => "135"+"0".repeat(24)
  * normalizeAmount 0.14 => "14"+"0".repeat(24-2)
  */
-export function normalizeAmount(value) {
+export function normalizeAmount(value, val) {
```

Comment:
1. Please rename `val` - it's confusing with `value`
2. Please add doc string about the  parameters.

Reply:
Ok